### ISSUE TITLE
RTSP and ONVIF not working

### ISSUE THREAD

**Issue by f-mns** [MuhammedKalkan/Anyka-Camera-Firmware#7]
Hello! I used your SD-Card method, but RTSP and ONVIF are not working. When i use telnet and execute the files I get errors:
```
[root@anyka /mnt/Factory/custom]$ ./rtsp 
default log level=4
[ak_vi_match_sensor:849] config_file: /etc/jffs2/
[ak_rtsp_vi_init:150] match sensor failed
vi init fail
```
```
[root@anyka /mnt/Factory/custom]$ ./onvif 
Segmentation fault
```
Inside /usr/modules:
![image](https://github.com/MuhammedKalkan/Anyka-Camera-Firmware/assets/78693466/fca1df4b-5078-41d1-ae92-98d0285e1103)

Inside /etc/jffs2:
![image](https://github.com/MuhammedKalkan/Anyka-Camera-Firmware/assets/78693466/e0ad66cf-9642-42ca-a91a-6a2e87fc2c27)
In sensor.tgz are `isp_gc1084.conf ` and `sensor_gc1084.ko`

I have this camera: https://de.aliexpress.com/item/1005005350475879.html?spm=a2g0o.productlist.main.1.2a95488eCgF8oE&algo_pvid=514b6fef-f92b-45ea-bb96-e7a6124da705&utparam-url=scene%3Asearch%7Cquery_from%3A
![IMG_9713](https://github.com/MuhammedKalkan/Anyka-Camera-Firmware/assets/78693466/677191b7-5cb0-4cb3-8dc5-32605890b579)

**Comment by MuhammedKalkan** [comment]
Your sensor driver is not inserted

You need to insert sensor_gc1084.ko too.

Exract it to sdcard and add this line on top of custom.sh

insmod sensor_gc1084.ko

**Comment by f-mns** [comment]
Thanks, but I now have another problem. I tried overwriting the camera software but now the camera is fried. It said "Update Complete", I disconnected the power, removed the sd card and when I connected power nothing happened. I connected it via serial, but every baud rate shows this:
![image](https://github.com/MuhammedKalkan/Anyka-Camera-Firmware/assets/78693466/2007876e-49fd-4bdd-959e-06a7b934cfd8)
Is there anything I can do or is the camera trash?

**Comment by f-mns** [comment]
No nothing. The Screenshot is with 115200 and I tried hours to get it work every connection should be right.

**Comment by MuhammedKalkan** [comment]
Then most likely connections. I dont think camera is trashed. Firmware only updates rootfs and usr.

Stock kernel is untouched, so your UART and all other drivers should be fine.

**Comment by f-mns** [comment]
What could I do now? I have backups from the folders in the tutorial.

**Comment by MuhammedKalkan** [comment]
1- Try to make serial work. This i would try rather than blindly updating

2- As a last resort, try to make custom firmware with your saved rootfs. Making a custom one is also explained

Did you try sdcard after flashing again?

**Comment by f-mns** [comment]
Yes, I connected the sd it made ptz calibratrion but no wifi. After a few tries nothing happend not even the config.sh is running on boot.

**Comment by MuhammedKalkan** [comment]
Ok then go for no 1 and 2 if 1 is absolutely not feasible

**Comment by f-mns** [comment]
When I try updating again there are no voice promps or any signs of life. How should I flash the custom firmware when I made it?

**Comment by MuhammedKalkan** [comment]
Hmm then no scripts are running. Only option is 1 then. Uboot should be running and serial sbould be up
